Composer/packages/client/src/pages/design/VisualEditor.tsx

```diff
@@ -105,6 +105,11 @@ const VisualEditor: React.FC<VisualEditorProps> = (props) => {
         aria-label={formatMessage('Visual editor')}
         css={visualEditor(triggerButtonVisible || !selected)}
         data-testid="VisualEditor"
+        onKeyDownCapture={(ev) => {
+          if (ev.key === 'Escape') {
+            (document.activeElement as HTMLElement)?.blur();
```

Comment:
The `document.activeElement` is not always an `HTMLDocument` e.g. it might be an `SVGElement`. That's why this cast is needed.